builder/role_image_test.go

```diff
@@ -120,10 +118,53 @@ func TestGenerateRoleImageRunScript(t *testing.T) {
 	runScriptContents, err = roleImageBuilder.generateRunScript(rolesManifest.Roles[1])
 	assert.NoError(err)
 	assert.NotContains(string(runScriptContents), "monit -vI")
-	assert.NotContains(string(runScriptContents), "/etc/monitrc")
```

Comment:
I changed the `run.sh` script to check if `/etc/monitrc` exists, and chmod if it does, so it's always going to have that text now.